libraries/radspreadprocessing/features/notes.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Notes
+page_title: Notes 
+slug: radspreadprocessing-features-notes
+tags: notes, xlsx
+published: True
+position: 23
+---
+
+
+# Notes
+
+Since R1 2022 the SpreadProcessing library supports working with notes. The Notes are used for making notes or annotations about the data. All notes can be found in the __NoteCollection__ of the worksheet. Each note has the following properties:
```

Comment:
I am not sure we are defining the specific release that the feature has been added.

Reply:
I believe it's better to have it since the client would not wonder why they do not have this when using an older version.

Reply:
I agree with you but until now I think we didn't do it this way and perhaps we should discuss this with Tanya or with the rest of the team.

Reply:
Ok, I will bring this question after the holidays.